CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-## 5.0.0-beta.0
+## 5.0.0
```

Comment:
Given the changelog entry is what many people will read first, can we just add the say 3 most important changes to it and link to respective docs? I think the specific imports for renderer and main process might be a candidate. wdyt?